feature/account/common/src/main/kotlin/app/k9mail/feature/account/common/ui/AccountTopHeader.kt

```diff
@@ -19,10 +18,10 @@ import app.k9mail.core.ui.compose.designsystem.atom.text.TextHeadline2
 import app.k9mail.core.ui.compose.designsystem.template.ResponsiveWidthContainer
 import app.k9mail.core.ui.compose.theme.MainTheme
 import app.k9mail.core.ui.compose.theme.PreviewWithThemes
-import app.k9mail.feature.account.setup.R.string
 
 @Composable
-internal fun AccountSetupTopHeader(
+fun AccountTopHeader(
```

Comment:
This isn't really related to accounts. Maybe `AppTitleTopHeader`, `IconTitleTopHeader`?